packages/camera/camera_android_camerax/lib/src/image_analysis.dart

```diff
@@ -49,18 +52,33 @@ class ImageAnalysis extends UseCase {
 
   late final _ImageAnalysisHostApiImpl _api;
 
+  /// Initial target rotation of the camera used for the preview stream.
+  ///
+  /// Should be specified in terms of one of the [Surface]
+  /// rotation constants that represents the counter-clockwise degrees of
+  /// rotation relative to [DeviceOrientation.portraitUp].
+  final int? initialTargetRotation;
```

Comment:
Reflecting on this now and wondering if we should even save the target rotation for `UseCase`s and instead expose methods like https://developer.android.com/reference/androidx/camera/core/ImageCapture#getTargetRotation() that allow us to retrieve the target rotation for each of them. (this is a side effect of these wrapper classes needing to be immutable)

@gmackall lmk what you think. I would probably do this in a follow up PR because this has a lot going on already. Running into a similar thing in the last wrappings I need to do and want to take a similar path.

Reply:
I support replacing with the native method, but am also fine with it being a follow up!

Reply:
Ok great! Filed https://github.com/flutter/flutter/issues/140664 to follow up.